pontoon/administration/files.py

```diff
@@ -770,9 +768,14 @@ def update_from_repository(project, locales=None):
         repository_url_master = repository_url.rsplit(ending, 1)[0]
         repository_path = os.path.join(repository_path_master, ending)
 
+    # If Mozilla repo, set paths
```

Comment:
Reading this more carefully, we pull the en-US version from your URL but then we pull the locale repositories one at a time. Gotcha.